backends/graphics/opengl/opengl-graphics.cpp

```diff
@@ -241,6 +241,7 @@ namespace {
 const OSystem::GraphicsMode glStretchModes[] = {
 	{"center", _s("Center"), STRETCH_CENTER},
 	{"pixel-perfect", _s("Pixel-perfect scaling"), STRETCH_INTEGRAL},
+	{"pixel-perfect-ar", _s("Pixel-perfect scaling AR"), STRETCH_INTEGRAL_AR},
```

Comment:
This requires an ```// I18N: AR means Aspect Ratio(?)``` comment for translators.

Reply:
As I noted in the PR description, this was a placeholder name as I could not find a good name for it. I have now changed it to "Even pixels scaling"